src/components/tag/tag/tag.scss

```diff
@@ -92,6 +92,11 @@
   }
 }
 
+:host([size='s']) {
+  --sbb-tag-height: #{sbb.px-to-rem-build(28)};
```

Comment:
@mcilurzo Should we maybe have a new size element token, or is this something completely different?
<img width="936" alt="Bildschirmfoto 2024-05-23 um 08 24 24" src="https://github.com/sbb-design-systems/lyne-components/assets/29278232/4ce34368-02d6-4e8c-a715-83ee418808be">

Reply:
i added a new size (xxs) with 28/32px size: https://www.figma.com/design/mWknI2rC5DJmOgRO61WKai/Lyne-Design-Tokens?node-id=2454-1578&t=P5VO8Sefrv4lPUcV-11